rumqttc/src/v5/notifier.rs

```diff
@@ -36,30 +34,34 @@ impl Notifier {
     pub fn iter(&mut self) -> NotifierIter<'_> {
         NotifierIter(self)
     }
+
+    #[inline]
+    /// Retruns None immediately even if not disconnected, use when expecting non-blocking interface
+    pub fn try_next(&mut self) -> Option<Incoming> {
```

Comment:
I suggest calling this `try_recv` instead, since `try_next` makes this sound like an iterator (which this struct is not).